lib/internal/Magento/Framework/Interception/Code/InterfaceValidator.php

```diff
@@ -71,11 +73,8 @@ public function validate($pluginClass, $interceptedType)
             $methodType = $this->getMethodType($pluginMethod->getName());
```

Comment:
It's pretty minor, but since we use the `if` statements instead of `switch` we may move the initialization of the following variables `$originMethod`,  `$originMethodParameters`, `$methodType` down to [line 85](https://github.com/magento/magento2/blob/25d0db9494c74009e0f9bdf1ccdbfca772669fd4/lib/internal/Magento/Framework/Interception/Code/InterfaceValidator.php#L85) (right before `if (self::METHOD_AFTER === $methodType && count($pluginMethodParameters) > 1) {`). Taking into account that these variables might be not used and it will also improve the readability a bit.

Reply:
Done, thank you